elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcClientSubsystemModel.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.elytron.oidc;
+
+import org.jboss.as.controller.ModelVersion;
+import org.jboss.as.controller.SubsystemModel;
+
+/**
+ * Enumeration of elytron-oidc-client subsystem model versions.
+ * @author <a href="mailto:prpaul@redhat.com">Prarthona Paul</a>
+ */
+
+enum ElytronOidcClientSubsystemModel implements SubsystemModel {
+    VERSION_1_0_0(1, 0, 0),
+    VERSION_2_0_0(2, 0, 0),
+    VERSION_3_0_0_COMMUNITY(3, 0, 0),
```

Comment:
Only management model changes to the default stability level require incrementing the subsystem model version.

Reply:
I will change it to just version 1 and 2. 
In that case, should I remove the following function? 
https://github.com/wildfly/wildfly/blob/89884940717ae10cf0e39689a19932607e978d6f/elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcSubsystemTransformers.java#L47-L51

Reply:
If the management model (for any stability level) changes, you still need to increment the model version.

Reply:
I have incremented the model version to 3 and included a from3(...) method